operations/kubernetes/Kyverno.md

```diff
@@ -456,3 +456,20 @@ and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/matc
 
     If the issue still persists after restarting the Kyverno pods, then HPE recommends disabling the background scanning in the Kyverno policy.
     Set `background` to `false` to disable background scanning in the Kyverno policy.
+* Multiple Kyverno admission reports may cause etcd downtime during upgrade to CSM 1.6.
```

Comment:
At what point during an upgrade can this happen? I assume there must be something in the upgrade which causes this, so we should at least be able to say that it would start being observed (for example) after CSM services are upgraded .